### ISSUE TITLE
the published version is incompatible w/ moment 2.19.0's module format

### ISSUE THREAD

**Issue by jsg2021** [moment/moment-timezone#540]
When importing as an ES module, the umd wrapper fails to account for the possible 'default' export name of moment.js (2.19 added the 'module' property to the package.json, and webpack follows it instead of main.)

**Comment by jsg2021** [comment]
on [moment-timezone.js#L14](https://github.com/moment/moment-timezone/blob/15d13436206c20042f238eaf6087735791295024/moment-timezone.js#L14):
```js
module.exports = factory(require('moment')); // Node
```
should add an interop function call between `factory()` and the return of `require()`

something like:
```
function interop (x) {
    return x && x.default || x;
}
```

**Comment by mj1856** [comment]
The was fixed automatically with the change in 2.19.1.   See also #541

**Comment by jsg2021** [comment]
Just clarifying... it does work now... however:

rollup and webpack (with [mainFields](https://webpack.js.org/configuration/resolve/#resolve-mainfields) set to look at `esnext:main` along with `module`, and `main`) will still hit this issue. If moment.js adds the `module` field back we will hit this problem.
